src/_includes/layout.njk

```diff
@@ -10,6 +10,8 @@
     <link rel="stylesheet" href="/static/css/reset.css" />
     <link rel="stylesheet" href="/static/css/variables.css" />
     <link rel="stylesheet" href="/static/css/utilities.css" />
+    <link rel="stylesheet" href="/static/css/link.css" />
+    <link rel="stylesheet" href="/static/css/typestack.css" />
```

Comment:
This is missing `font.css` which is why the fonts aren't appearing properly! Can you put it right after `variables.css`?